db/migrate/20250708131033_create_instructions.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class CreateInstructions < ActiveRecord::Migration[8.0]
+  def change
+    create_table :instructions do |t|
+      t.references :created_by_user,
+                   null: false,
+                   foreign_key: {
+                     to_table: :users
+                   }
+      t.references :patient, null: false, foreign_key: true
+      t.references :programme, null: false, foreign_key: true
+      t.references :vaccine, null: false, foreign_key: true
+
+      t.string :vaccine_method, null: false
+      t.string :delivery_site, null: false
+      t.boolean :full_dose, null: false
+      t.string :protocol, null: false
```

Comment:
I think `vaccine_method` and `delivery_site` should probably be enums (and therefore an integer) and with the same enum values as what we have already for triage status and vaccination records.

I think it would probably also make sense for `protocol` to be an integer-backed enum too.